src/blocks_css/general.css

```diff
@@ -0,0 +1,16 @@
+@import url(./normalize.css);
+@import url(./cards.css);
+@import url(./stars.css);
+
```

Comment:
<img width="363" alt="Screenshot 2023-01-27 at 10 54 35" src="https://user-images.githubusercontent.com/94289038/215046507-d27dcb45-18a8-4aaf-bcce-1e1f92c0187b.png">
looks like it should be `index.css`